src/main/kotlin/io/libp2p/pubsub/gossip/Gossip.kt

```diff
@@ -24,10 +23,6 @@ class Gossip @JvmOverloads constructor(
         router.score.updateTopicParams(scoreParams)
     }
 
-    fun getGossipScore(peerId: PeerId): GossipScore.PeerScores {
```

Comment:
I think we still want to expose this from `Gossip`

Reply:
I see, the API will be exported from Gossip.